README.md

```diff
@@ -1,2 +1,94 @@
-# janitor
+<!-- omit from toc -->
+# Janitor
 A Python application which hosts task scripts for data management and clean up tasks, running them on a schedule.
+The application uses APScheduler to schedule and run these tasks.
+
+<!-- omit from toc -->
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Requirements for Development](#requirements-for-development)
+  - [Installing Dependencies](#installing-dependencies)
+  - [Configuring the Environment](#configuring-the-environment)
+- [Running the Application](#running-the-application)
+  - [Adding Tasks](#adding-tasks)
+- [Testing](#testing)
+- [Formatting, Type Checking and Linting](#formatting-type-checking-and-linting)
+- [Deployment](#deployment)
+
+
+## Getting Started
+
+### Requirements for Development
+
+The following tools are required for development:
+
+- Python (use a tool like `pyenv` to install the required version specified in the `Pipfile`)
+- Install dependencies using [pipenv](https://github.com/pypa/pipenv):
+
+        brew install pipenv
+
+### Installing Dependencies
+
+Install the required dependencies:
+
+    pipenv install
+    pipenv install --dev
+
+### Configuring the Environment
+
+The app can be configured to run with development settings by adding
+
+    SETTINGS_MODULE=janitor.config.development
+
+to a `.env` file.
```

Comment:
I think it would be better if you create a `.env.example` file in the repo with this key set and change your instructions here to renaming the file to `.env` as part of the repo setup.  Everyone should be using the `development` module under normal conditions.

Reply:
I agree, a better approach would be to rename a `.env.example` file instead of creating one manually so I've added instructions for that too